pkgmgmtutils/internal/formula_bytes_updater_test.go

```diff
@@ -54,4 +56,46 @@ var _ = Describe("FormulaBytesUpdater", func() {
 			)
 		}).To(Panic())
 	})
+
+	It("can replace homebrew-core version and sha", func() {
+		testVersion := "v7.8.9"
+		testVersionSha := "gitcommitsha"
+		formula := formula_updater_types.FormulaOptions{
+				Name:            "homebrew-core/glooctl",
+				FormulaName:     "glooctl",
+				Path:            "Formula/glooctl.rb",
+				RepoOwner:       "solo-io",
+				RepoName:        "homebrew-core",
+				PRRepoOwner:     "homebrew",
+				PRRepoName:      "homebrew-core",
+				PRBranch:        "master",
+				PRDescription:   "Created by Solo-io Bot",
+				PRCommitName:    "Solo-io Bot",
+				PRCommitEmail:   "bot@solo.io",
+				VersionRegex:    `tag:\s*"v([0-9.]+)",`,
+				VersionShaRegex: `revision:\s*"(.*)"`,
+		}
+
+		// GitHub API docs: https://developer.github.com/v3/repos/contents/#get-contents
+		gclient := github.NewClient(nil)
+		ctx := context.Background()
+		fileContent, _, _, err := gclient.Repositories.GetContents(ctx, formula.RepoOwner, formula.RepoName, formula.Path, &github.RepositoryContentGetOptions{
+			Ref: "refs/heads/master",
+		})
+		Expect(err).To(Not(HaveOccurred()))
+
+		c, err := fileContent.GetContent()
+		Expect(err).To(Not(HaveOccurred()))
```

Comment:
nit: are there any github utils that we could use to achieve this?

Reply:
It looks like GetFilesFromGit can be used instead!